test/integration/scheduler_perf/test-performance.sh

```diff
@@ -48,5 +48,6 @@ if ${RUN_BENCHMARK:-false}; then
 fi
 # Running density tests. It might take a long time.
 kube::log::status "performance test (density) start"
-go test -test.run=. -test.timeout=60m -test.short=false
+go test -test.timeout=60m -test.short=false -test.run=.
```

Comment:
Nothing. I was testing with few other changes and this got pushed. No need to touch this file as such. I will remove it.